interfaces/seccomp/backend_test.go

```diff
@@ -736,3 +737,75 @@ func (s *backendSuite) TestCompilerInitUnhappy(c *C) {
 	err := s.Backend.Initialize()
 	c.Assert(err, ErrorMatches, "cannot initialize seccomp profile compiler: failed")
 }
+
+func (s *backendSuite) TestSystemUsernamesPolicy(c *C) {
+	snapYaml := `
+name: app
+version: 0.1
+system-usernames:
+  testid: shared
```

Comment:
We *may* want to add two test users and check that both get added to the profile.